src/ansys/aedt/core/generic/file_utils.py

```diff
@@ -948,11 +948,17 @@ def available_license_feature(
             port = name_env[0]
             name = name_env[1]
 
-    if not input_dir:
+    if not input_dir and aedt_versions.latest_version:
```

Comment:
@maxcapodi78 This logic is different than what you proposed. This appears to work for our use case, but is it ok to prefer the latest_version over current_version as we do here?

Reply:
I think it's best to favor the latest_version for licensing so this makes sense to me. The tests pass and this fixed the issue we were seeing locally for 25.2.